### ISSUE TITLE
Upgrade from C129 -> C130 on Win x64 via 1.72.x - Nightly (Checking C130)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#41323]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [ ] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [ ] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [ ] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [ ] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [ ] `CertificateRevocation`
  - [ ] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [ ] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [ ] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [ ] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [ ] `Safe Browsing`
- [ ] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [ ] Make sure that data from the last version appears in the new version OK
- [ ] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [ ] With data from the last version, verify that:
  - [ ] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [ ] Cookies are preserved
  - [ ] Installed extensions are retained and work correctly
  - [ ] Opened tabs can be reloaded
  - [ ] Stored passwords are preserved
  - [ ] Sync chain created in previous version is retained
  - [ ] Social media-blocking buttons changes are retained
  - [ ] Custom filters under brave://settings/shields/filters are retained
  - [ ] Custom lists under brave://settings/shields/filters are retained
  - [ ] Rewards
    - [ ] BAT balance is retained
    - [ ] Auto-contribute list is retained
    - [ ] Both Tips and Monthly Contributions are retained
    - [ ] Panel transactions list is retained
    - [ ] Changes to rewards settings are retained
    - [ ] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [ ] Ads
    - [ ] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [ ] Changes to ads settings are retained
    - [ ] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [ ] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave | 1.72.48 Chromium: 130.0.6723.19 (Official Build) nightly (64-bit)
-- | --
Revision | 70b441e85b783a65b0088d92c496fa56d6509945
OS | Windows 11 Version 23H2 (Build 22631.4169)
```

<details>
<summary>C129 --> C130 on-demand upgrade check - Pending</summary>

1. installed latest nightly ` `  and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6. ensured Brave was updated to `  ` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 6`**
----------------|----------------|----------------|---------------|------------


</details>

<details>
<summary>C129 --> C130 background upgrade check - Pending</summary>

1. installed latest nightly ` ` and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
-   * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. delete any `last checked` entry
5. started `BraveSoftwareUpdateTaskMachineUA{FCFE660D-D369-4717-9908-83F75DF38808}` via `Task Scheduler`
6. waited ~30 min or so before launching Brave and ensured it was updated to ` `  (removed `last checked` in `BraveSoftware\Update\` 
7. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|---------------|----------------|-----------



</details>

<details>
<summary>C130--> C130 on-demand upgrade check - Pending</summary>

1. installed `1.72.48 Chromium: 130.0.6723.19 (Official Build) nightly (64-bit)` 
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:

- * `https://updates.brave.com/service/update2`

3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. visited `brave://settings/help` and ensured that new update was being pulled/downloaded
5. ensured that clicking `Relaunch` restarted the browser without any issues
6. ensured Brave was updated to `   ` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|----------------|---------------|----------


</details>

<details>
<summary>C130 --> C130  background upgrade check - Pending</summary>

1. installed `1.72.48 Chromium: 130.0.6723.19 (Official Build) nightly (64-bit)` and closed Brave
2.  created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
    * `https://updates.brave.com/service/update2`
3.  set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
4. delete any `last checked` entry
5. started `BraveSoftwareUpdateTaskMachineUA{FCFE660D-D369-4717-9908-83F75DF38808}` via `Task Scheduler`
6. closed Brave
7. waited ~30min or so before launching Brave and ensured it was updated to `    `
8. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 7`**  
----------------|----------------|------------|------------|--------- 




</details>

**Comment by LaurenWags** [comment]
Verification in progress on `Win 10 x86` using the following build(s):

```

```

<details>
<summary>C129 --> C130 on-demand upgrade check - InProgress</summary>

* installed `1.72.48` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.71.75 Chromium: 129.0.6668.31` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|---------------

</details>
